app/services/catalog/submit_order.rb

```diff
@@ -13,7 +12,6 @@ def process
       @order = Order.find_by!(:id => @order_id)
       @order.order_items.each do |order_item|
         raise Catalog::NotAuthorized unless valid_source?(order_item.portfolio_item.service_offering_source_ref)
-        raise Catalog::InvalidSurvey, "Base survey does not match Topology" if item_surveys_changed?(order_item.portfolio_item)
```

Comment:
Removing this check here, since we already checked for it before submitting the order. 

If we should keep this, let me know I can add it back in. I'm not sure we're that worried about it changing after the successful submit, since minion will just submit it silently and we won't see the exception from the frontend. Maybe we could log a progress message or something?

Reply:
Ended up talking with @syncrou about this, decided that this is an edge case where we can fail the order item and log a progress message. That way if this ever did happen the UI would definitely know it.